app/src/main/java/org/covidwatch/android/ui/home/HomeFragment.kt

```diff
@@ -146,4 +155,26 @@ class HomeFragment : Fragment() {
         binding.testedButton.isVisible = false
         binding.testedButtonText.isVisible = false
     }
+
+    private fun getFirebaseIdIfTester() {
+        val context = requireContext()
+        val packageManager: PackageManager = requireContext().packageManager
+        val componentName: ComponentName = ComponentName(context, MainActivity::class.java)
+        val ai: ActivityInfo = packageManager.getActivityInfo(componentName, PackageManager.GET_META_DATA)
+        val metaData = ai.metaData
+        if (metaData == null || metaData["firebaseDebugging"] != true) {
+            binding.testerId.visibility = View.GONE
+            setTester(false)
+            return;
+        } else {
+            val firebaseId: String = getFirebaseId()
+            binding.testerId.setText("Your Firebase tester id is: " + firebaseId)
```

Comment:
nit: Can we change the text from `"Your Firebase tester id is: "` to `"Your COVID Watch Tester Id is: "`? Testers won't know what Firebase is